contracts/SpecificGroupStrategy.sol

```diff
@@ -33,12 +33,25 @@ contract SpecificGroupStrategy is UUPSOwnableUpgradeable, UsingRegistryUpgradeab
      * @notice stCELO that was cast for specific group strategies,
      * strategy => stCELO amount
      */
-    mapping(address => uint256) public stCeloInGroup;
+    mapping(address => uint256) public stCeloInStrategy;
 
     /**
-     * @notice Total stCELO that was voted with on specific group strategies.
+     * @notice Total stCELO that was voted with on specific group strategies (including overflows).
+     * @dev To get the actual stCelo in specific strategy
+     * it is necessary to subtract `totalStCeloOverflow`.
      */
-    uint256 public totalStCeloInStrategy;
+    uint256 public totalStCeloLocked;
+
+    /**
+     * @notice stCELO that was cast for specific group strategies and overflown to defautl strategy,
```

Comment:
Grammar nit: I *think* this should be "overflowed", not "overflown", here and probably most other cases throughout.